Esmorga-Backend/esmorga-backend
# [MOB-134] Add refreshToken string validation and new logic to the login

## Pull Request Checklist

- [x] **[MOB US' Link](https://mobilemakers.atlassian.net/browse/MOB-134)**

## Testing Guide
Check the new refreshToken validation and that the login attempts document is removed after a successful login.

**Login:**
```
curl --location 'http://localhost:3000/v1/account/login' \
--header 'Content-Type: application/json' \
--data-raw '{
    "email": "login@yopmail.com",
    "password": "SuperSecret1!"
}'
```
**Refresh token:**
```
curl --location 'http://localhost:3000/v1/account/refresh' \
--header 'Content-Type: application/json' \
--data '{
    "refreshToken": "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1dWlkIjoiNjZmZDMyYTNlMmY4MjE0NmI4ZDRmZTk0IiwiaWF0IjoxNzI4Mjg4NTcwfQ.BXlKwKaqLROqC5LqKgI6xKq1Ui3AYeBKxdpkw_vQP1A"
}'
```
